src/test/java/com/microsoft/sqlserver/jdbc/connection/ConnectionTest.java

```diff
@@ -43,4 +43,32 @@ public void testConnections() throws SQLException {
         try (Connection con = ds.getConnection()) {
         }
     }
+    
+    @Test
+    public void testConnectWithIPAddressPreference () throws SQLException {
+        SQLServerDataSource ds = new SQLServerDataSource();
+        ds.setURL(connectionString);
+        ds.setIPAddressPreference("IPv4First");
+        
+        try (Connection con = ds.getConnection()) {

```

Comment:
[nitpick] there seems to be a lot of unnecessary/random spacing here can we group the tests together so it's easier to read?

Reply:
Changes made as asked.